docs/examples/index.md

```diff
@@ -1,8 +1,137 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: 'Code examples'
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+## Passing an input to the Actor
 
-There is nothing here yet.
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+Input can be set up, can be passed to `call` function and the reference of running Actor (or wait for finish) is available in `runData` variable.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+const input = { hashtags: ['rainbow'], resultsLimit: 20 };
+
+// Run the Actor and wait for it to finish up to 60 seconds.
+// Input is not persisted for next runs.
+const runData = await actorClient.call(input, { waitSecs: 60 });
+```
+
+To run multiple inputs with the same Actor, most convenient way is to create multiple [tasks](https://docs.apify.com/platform/actors/running/tasks) with different inputs. Task input is persisted on Apify platform when task is created.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const animalsHashtags = ['zebra', 'lion', 'hippo'];
+
+// Multiple input schemas for one Actor can be persisted in tasks.
+// Tasks are saved in the Apify platform and can be run multiple times.
+const socialsTasksPromises = animalsHashtags.map((hashtag) => client.tasks().create({
+    actId: 'apify/instagram-hashtag-scraper',
+    name: `hashtags-${hashtag}`,
+    input: { hashtags: [hashtag], resultsLimit: 20 },
+    options: { memoryMbytes: 1024 },
+}));
+
+// Create all tasks in parallel
+const createdTasks = await Promise.all(socialsTasksPromises);
+
+// Run all tasks in parallel
+await Promise.all(createdTasks.map((task) => client.task(task.id).call()));
```

Comment:
Let's add some console logs here too - same reasoning as the comment above

Reply:
Please check lines below where you can find additional content